modular_citadel/code/datums/status_effects/debuffs.dm

```diff
@@ -5,7 +5,7 @@
 			buckl.unbuckle_mob(new_owner)
 		new_owner.resting = TRUE
 		new_owner.adjustStaminaLoss(isnull(override_stam)? set_duration*0.25 : override_stam)
-		if(isnull(override_duration) && (set_duration > 80))
+		if(isnull(override_duration) && (set_duration >= 80))
```

Comment:
Changing the default hardstun threshold from > 80 to >= 80 affects a *lot* of knockdown sources in the game, the use of > instead of >= absolutely is not a mistake. Sleeping carp, soap, and more are **heavily** buffed by this change. If you want a given knockdown source to force the hardstun regardless of length, use the override_hardstun arg and set it to something small like 0.1, instead of buffing a good chunk of stun sources in the game.

Reply:
The results of such change would actually be of a very marginal disarm.
Many of these sources call Stun(), drop_all_held_items() or accident() already.
While others are not really related stun-combat or are actually drawbacks or niche side-efects.
And Sleeping Carp moves are nerfed to hell right now, having shoddy knockdowns with no extra stamina damage or disarm.
Regardless of opinions, I admit I did not clearly took in consideration its offtopic consequences, so I'll do a 360° to keep this atomized.